### PR TITLE
p7zip: Update package

### PR DESCRIPTION
**Pull request by kenneth-Q**
Add p7zip and p7zip-rar
p7zip up to date

### CODE REVIEW

**Comment by xeffyr** [comment]
Rar codec is **already available** !

Check package with `dpkg -L p7zip`.

**Comment by xeffyr** [comment]
What you can to do is only apply security patches to package **p7zip**.

File: packages/jp2a/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/cslarsen/jp2a
+TERMUX_PKG_DESCRIPTION="a simple JPEG to ASCII converter."
```
Comment (xeffyr) [inline packages/jp2a/build.sh]: And what this is doing here ?

Comment (xeffyr) [inline packages/jp2a/build.sh]: What this is doing here ?

Comment (kenneth-Q) [reply packages/jp2a/build.sh]: This package just for conversion.
jpeg file to ascii art

Comment (kenneth-Q) [reply packages/jp2a/build.sh]: Just for fun :D

**Pull request closed without merging**